src/installer.js

```diff
@@ -17,12 +17,14 @@ const defaultOptions = {
   dependencies: {
     peer: true,
   },
-  packageManagerOptions: {
-    dev: false,
+  packageManager: {
+    type: 'npm',
```

Comment:
We should use `getDefaultPackageManager` here we do here https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/get-package-manager.js (in separate PR, it is feature)